web/src/test/java/com/epam/rft/atsy/cucumber/util/CommonStepDefs.java

```diff
@@ -16,4 +27,73 @@ public void loggedIn() {
     getDriver().findElement(By.id("password")).sendKeys("pass3");
     waitForPageLoadAfter(event -> getDriver().findElement(By.id("loginButton")).click());
   }
+
+  @When("^the user clicks on (.*) in the header$")
+  public void theUserClicksOnButtonInTheHeader(String button) throws Throwable {
+      switch (button) {
+          case "epam-logo":
+              driverProvider.getDriver().findElement(By.cssSelector("img.img-rounded")).click();
+              break;
+          case "Beállítások-icon":
+              driverProvider.getDriver().findElement(By.id("settings_link")).click();
+              break;
+          case "Beállítások-text":
+              driverProvider.getDriver().findElement(By.linkText("Beállítások")).click();
+              break;
+          case "Kilépés-icon":
+              driverProvider.getDriver().findElement(By.id("logout_link")).click();
+              break;
+          case "Kilépés-text":
+              driverProvider.getDriver().findElement(By.linkText("Kilépés")).click();
+              break;
+      }
+
+  }
+
+  @Then("^the user should get epam logo in the header$")
+  public void theUserGetsEpamLogoInTheHeader() throws Throwable {
+      WebElement element = driverProvider.getDriver().findElement(By.cssSelector("img.img-rounded"));
+      assertThat(element.isDisplayed(), is(true));
+  }
+
+  @Then("^the user should get Beállítások link in the header$")
+  public void theUserGetsSettingsLinkInTheHeader() throws Throwable {
+      WebElement text_element = driverProvider.getDriver().findElement(By.linkText("Beállítások"));
+      assertThat(text_element.getText(), is("Beállítások"));
+      assertThat(text_element.isDisplayed(), is(true));
+      WebElement icon_element = driverProvider.getDriver().findElement(By.id("settings_link"));
+      assertThat(icon_element.isDisplayed(), is(true));
+  }
+
+  @Then("^the user should get Kilépés link in the header$")
+  public void theUserGetsLogoutLinkInTheHeader() throws Throwable {
+     WebElement text_element = driverProvider.getDriver().findElement(By.linkText("Kilépés"));
+     assertThat(text_element.getText(), is("Kilépés"));
+     assertThat(text_element.isDisplayed(), is(true));
+     WebElement icon_element = driverProvider.getDriver().findElement(By.id("logout_link"));
+     assertThat(icon_element.isDisplayed(), is(true));
+  }
+
+  @Then("^(.*) url should open$")
+  public void UrlOpens(String page) throws Throwable {
```

Comment:
The java naming convention is to use camelcase for method names, but start with a lowercase letter.

In this example, the name for this method should look like: `urlOpens`.